flink-connectors/flink-connector-aws-base/src/main/java/org/apache/flink/connector/aws/util/AWSGeneralUtil.java

```diff
@@ -369,4 +367,35 @@ public static void closeResources(SdkAutoCloseable... resources) {
             throw exception;
         }
     }
+
+    public static void validateWebIdentityTokenFileCredentialsProvider(Properties config) {
+        validateCredentialProvider(config);
+        try {
+            CredentialProvider credentialProviderType =
+                    getCredentialProviderType(config, AWSConfigConstants.AWS_CREDENTIALS_PROVIDER);
+            if (credentialProviderType.equals(WEB_IDENTITY_TOKEN)) {
+                getCredentialsProvider(config).resolveCredentials();
+            }
+        } catch (Throwable e) {
```

Comment:
Can you catch something more specific that Throwable here? Why catch throwable? This is usually considered bad practise